### PR TITLE
Added new OutwardArrowsIcon

### LINKED ISSUE #708: New Width and Height icons
Add the attached icons into our icon library.
[Width-Height_Icons.zip](https://github.com/appnexus/lucid/files/761698/Width-Height_Icons.zip)

### PR DESCRIPTION
**Pull request by jondlm**
Fixes #708

http://docspot.devnxs.net/projects/lucid/708-icons/#/components/OutwardArrowsIcon

## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [ ] Firefox
  - [ ] Safari
  - [ ] IE11 (Win 7)
  - [ ] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [ ] One core team UX approval

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -49,7 +49,6 @@ import CrossIcon from './components/Icon/CrossIcon/CrossIcon';
 import DangerIcon from './components/Icon/DangerIcon/DangerIcon';
 import DangerLightIcon from './components/Icon/DangerLightIcon/DangerLightIcon';
 import DataTable from './components/DataTable/DataTable';
-import EmptyStateWrapper from './components/EmptyStateWrapper/EmptyStateWrapper';
```
Comment (jondlm) [inline src/index.js]: Just sorting alphabetically

File: src/components/Icon/OutwardArrowsIcon/OutwardArrowsIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Icon from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { createClass } from '../../../util/component-types';
+
+const { oneOf } = React.PropTypes;
+
+const cx = lucidClassNames.bind('&-OutwardArrowsIcon');
+
+const paths = {
```
Comment (jondlm) [inline src/components/Icon/OutwardArrowsIcon/OutwardArrowsIcon.jsx]: I created a single icon that supports all three rotations

Comment (sodiumjoe) [inline src/components/Icon/OutwardArrowsIcon/OutwardArrowsIcon.jsx]: will there never be a use case for diagonal the other way?

Comment (jondlm) [inline src/components/Icon/OutwardArrowsIcon/OutwardArrowsIcon.jsx]: I don't think so, it could be `diagonal-inverted` 🚎 

But yeah, I don't think so.

Comment (sodiumjoe) [inline src/components/Icon/OutwardArrowsIcon/OutwardArrowsIcon.jsx]: should be `oneOf(['horizontal', 'vertical', '/', '\'])`

**Pull request merged**